ws-nextjs-app/pages/[service]/new_live/[id]/live.test.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {
+  render,
+  screen,
+  act,
+} from '#app/components/react-testing-library-with-providers';
+import Live from './LivePageLayout';
+
+const mockPageData = {
+  pageCount: 10,
+  activePage: 1,
+  someResponse: {
+    block: 'Its a block',
+  },
+};
+
+describe('Live Page', () => {
+  it('renders the live page', async () => {
+    await act(async () => {
+      render(<Live pageData={mockPageData} />);
+    });
```

Comment:
Feels like I shouldn't need to wrap this in `await act(...` The Express app page components don't need it, so its possible a Next.JS thing.